client/state/login/actions.js

```diff
@@ -137,12 +137,11 @@ export const loginUser = ( usernameOrEmail, password, rememberMe ) => dispatch =
 /**
  * Attempt to login a user when a two factor verification code is sent.
  *
- * @param  {String}    two_step_code  Verification code for the user.
- * @param  {Boolean}   remember_me       Flag for remembering the user for a while after logging in.
+ * @param  {String}    twoStepCode  Verification code for the user.
  * @param {String}     twoFactorAuthType Two factor authentication method
  * @return {Function}                 Action thunk to trigger the login process.
  */
-export const loginUserWithTwoFactorVerificationCode = ( two_step_code, remember_me, twoFactorAuthType ) => ( dispatch, getState ) => {
+export const loginUserWithTwoFactorVerificationCode = ( twoStepCode, twoFactorAuthType ) => ( dispatch, getState ) => {
 	dispatch( { type: TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST } );
```

Comment:
With the current logic you are right, but in a broad sense it's risky as Redux state can diverge from component had stored when an action was triggered.